backend/app/assets/javascripts/spree/backend/global/animate_css.es6

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+
+//
+// Handle clearing out of animation styles from complete animation.
+const animateCSS = (element, animation, speed, prefix = 'animate__') =>
```

Comment:
The `prefix` variable is never being passed in from any of the places where we're calling this method. I'd argue for simplicity of the API and remove the unnecessary argument until you don't really need it in the future. If you don't need it now, don't add it. I'd simply have the first line of this method as:

```js
const prefix = 'animate__'
```

Or just inline the variable wherever it's used below:

```js
    const animationName = `animate__${animation}`
```